Gems/EMotionFX/Code/Source/Editor/Plugins/SkeletonOutliner/SkeletonOutlinerPlugin.h

```diff
@@ -80,11 +80,15 @@ namespace EMotionFX
         SkeletonSortFilterProxyModel*           m_filterProxyModel;
         static constexpr int s_iconSize = 16;
 
-        JointPropertyWidget* m_propertyWidget = nullptr;
         // Callbacks
         // Works for all commands that use the actor id as well as the joint name mixins
         MCORE_DEFINECOMMANDCALLBACK(DataChangedCallback);
         static bool DataChanged(AZ::u32 actorId, const AZStd::string& jointName);
         AZStd::vector<MCore::Command::Callback*> m_commandCallbacks;
+
+    protected:
+        JointPropertyWidget* m_propertyWidget = nullptr;
+
+        friend class SkeletonOutlinerTestFixture;
```

Comment:
Why add SkeletonOutlinerTestFixture as the friend class?
The UI class shouldn't know the TestFixture exist. Is it because the TestFixture need to access the private member of the plugin?

Reply:
Tom asked me to just make this public. Somehow that change got lost. Will make it public again.

Reply:
Yeah I just think `friend` relationships tend to be bad in general as they break encapsulation and lead to hard to maintain dependencies. By improving the interface or breaking things up we can get much better designs in the process :+1: